test/gurka/test_landmarks.cc

```diff
@@ -230,15 +208,16 @@ TEST(LandmarkTest, TestTileStoreLandmarks) {
       EXPECT_EQ(landmark.id, 0);
       EXPECT_EQ(landmark.name, std::to_string(edge_index++));
       EXPECT_EQ(landmark.type, ltype);
-      EXPECT_EQ(round_to(landmark.lng), round_to(point.first));
-      EXPECT_EQ(round_to(landmark.lat), round_to(point.second));
+      EXPECT_NEAR(landmark.lng, point.first, rounding_error);
+      EXPECT_NEAR(landmark.lat, point.second, rounding_error);
 
       // check data correctness of the fixed landmark
+
       // EXPECT_EQ(landmark.id, 0);
       // EXPECT_EQ(landmark.name, "fixed landmark");
       // EXPECT_EQ(landmark.type, LandmarkType::casino);
-      // EXPECT_EQ(round_to(landmark.lng), round_to(lng));
-      // EXPECT_EQ(round_to(landmark.lat), round_to(lat));
+      // EXPECT_NEAR(landmark.lng, lng, rounding_error);
+      // EXPECT_NEAR(landmark.lat, lat, rounding_error);
```

Comment:
this should be part of the cleanup IMO. also further down. sorry for the nits, but without any comment why we keep commented code in tests, it only adds confusion. if there is a reason, best to leave to a comment about it

Reply:
https://github.com/valhalla/valhalla/pull/4233/files#diff-6b826629e0424f344205ddbff8872b3d9a1c3bd856ca21cece6d226656ab0aebR164
there is small comment on this - we can either try to test the fixed landmark or flexible ones. I tried to keep both but I guess we have the flexible landmarks and there is no need to keep the fixed one :) already cleaned up.